tests/test_playbooks/tasks/wait_for_task.yml

```diff
@@ -41,8 +39,7 @@
     validate_certs: "{{ foreman_validate_certs }}"
     task: "{{ item }}"
     timeout: 900
-  loop: "{{ tasks.resources }}"
-
+  loop: "{{ tasks.resources | map(attribute='id') | list }}"
```

Comment:
Just to confirm, we could also have used `{{ item.id }}`?

Reply:
sure, but where would be the fun.

my brainfart, sorry. :)

Reply:
on the other hand, this notation makes the ansible output nicer, as it only shows the ids and not ALL the things we got from `resource_info`.

Reply:
^ That's what i thought when i was about to make this a suggestion.

Reply:
And there is also
```yaml
loop_control:
  label: "{{ item.name }}:{{ item.id }}"
```
If you want _nice_.

Reply:
modern shit :D